### PR TITLE
Add search by plume id and by location

### PR DESCRIPTION
**Pull request by amarouane-ABDELHAK**
Add search by plume id and by location

### CODE REVIEW

File: emit-ch4plume-v1/update_data.js [hunk 1]
```diff
@@ -77,6 +134,39 @@ fs.writeFileSync(
...
+
+  //   dict['key'] = "testing";
+    
+  //   console.log(dict);
+  
+//   console.log(lookup_location['CH4_PlumeComplex-2296']);
+
+// }
+
+// create_lookup()
```
Comment (slesaad) [inline emit-ch4plume-v1/update_data.js:172]: 👀

File: emit-ch4plume-v1/src/style.css [hunk 2]
```diff
@@ -201,3 +202,83 @@ button.mapboxgl-ctrl-zoom-out .mapboxgl-ctrl-icon {
...
+
+.autocomplete-search-box .search-result {
+  background-color: white;
+  max-height: 200px;
+  overflow: auto;
+}
+
+
+
+
```
Comment (slesaad) [inline emit-ch4plume-v1/src/style.css:284]: signature @amarouane-ABDELHAK newlines 😛

File: emit-ch4plume-v1/update_data.js [hunk 3]
```diff
@@ -33,11 +34,65 @@ there_is_more_data = false;
...
+  if (change_lookup) {
+    fs.writeFileSync(
+      "./lookups/plume_id_location.json",
+      JSON.stringify(lookup_location, null, 2)
+      );
+
+  }
+
+
+
```
Comment (slesaad) [inline emit-ch4plume-v1/update_data.js:89]: 👀

File: emit-ch4plume-v1/src/style.css (hunk 2 above)
Comment (amarouane-ABDELHAK) [reply emit-ch4plume-v1/src/style.css:284]: linter ugggggh

File: emit-ch4plume-v1/update_data.js (hunk 1 above)
Comment (amarouane-ABDELHAK) [reply emit-ch4plume-v1/update_data.js]: fixed

File: emit-ch4plume-v1/src/style.css (hunk 2 above)
Comment (amarouane-ABDELHAK) [reply emit-ch4plume-v1/src/style.css]: fixed

File: emit-ch4plume-v1/update_data.js (hunk 3 above)
Comment (amarouane-ABDELHAK) [reply emit-ch4plume-v1/update_data.js]: fixed

**Review by slesaad — APPROVED** [review]

**Pull request merged**